test/index.js

```diff
@@ -120,21 +120,24 @@ describe('preBuild()', () => {
     expect(await pathExists('next.config.js')).toBeTruthy()
   })
 
-  test(`fail build if the app's next config has an invalid target`, async () => {
-    const { restoreCwd } = useFixture('invalid_next_config')
-    await plugin.onPreBuild({
-      netlifyConfig: {},
-      packageJson: DUMMY_PACKAGE_JSON,
-      utils,
-      constants: { FUNCTIONS_SRC: 'out_functions' },
-    })
-    restoreCwd()
-
-    const acceptableTargets = ['serverless', 'experimental-serverless-trace']
-    expect(utils.build.failBuild.mock.calls[0][0]).toEqual(
-      `next.config.js must be one of: ${acceptableTargets.join(', ')}`,
-    )
-  })
+  test.each(['invalid_next_config', 'deep_invalid_next_config'])(
+    `fail build if the app's next config has an invalid target`,
+    async (fixtureName) => {
+      const { restoreCwd } = useFixture(fixtureName)
+      await plugin.onPreBuild({
+        netlifyConfig: {},
+        packageJson: DUMMY_PACKAGE_JSON,
+        utils,
+        constants: { FUNCTIONS_SRC: 'out_functions' },
+      })
+      restoreCwd()
+
+      const acceptableTargets = ['serverless', 'experimental-serverless-trace']
+      expect(utils.build.failBuild.mock.calls[0][0]).toEqual(
+        `next.config.js must be one of: ${acceptableTargets.join(', ')}`,
+      )
+    },
```

Comment:
friendly reminder to update this with the new fixture and failBuild logic whenever you merge everything :) <3

Reply:
Rebased and fixed! :+1: